src/libgpu/src/glsl2/glsl2_tex.cpp

```diff
@@ -181,8 +187,9 @@ sampleFunc(State &state,
          decaf_check(func.size());
          state.out << func;
       }
-
-      state.out << "(sampler_" << samplerID << ", ";
+	  
+	  //Here we should not use samplerID
```

Comment:
* Maybe you should explain you don't use the samplerID here as well.
* Indent with 1 more space
* A space after the `//` after every one of your comments wouldn't hurt either.